src/pages/EventDetailsPage/components/EventDetails/index.tsx

```diff
@@ -72,12 +84,8 @@ function EventDetailsComponent(props: EventDetailsComponentProps) {
 
                     <Grid item>
                       <Grid container justify='flex-end' spacing={1}>
-                        <Grid item>
-                          <SignInButton eventId={eventId} />
-                        </Grid>
-                        <Grid item>
-                          <RSVPButton eventId={eventId} />
-                        </Grid>
+                        <Grid item>{renderSignInButton()}</Grid>
```

Comment:
probably don't need to have 2 extra functions here, just add a `status === EventStatusEnum.Ready` ? <Grid ...> : <></>